code/go/0chain.net/smartcontract/storagesc/stakepool.go

```
@@ -425,6 +425,13 @@ func (ssc *StorageSmartContract) stakePoolUnlock(
 			return "", common.NewErrorf("stake_pool_unlock_failed",
 				"saving stake pool: %v", err)
 		}
+		data, _ := json.Marshal(event.DbUpdates{
```

Please do not ignore the error as we don't know what would happen, and if error happens, we should avoid the empty `data` going into event db.